opm/simulators/linalg/bda/BdaBridge.cpp

```diff
@@ -216,7 +217,7 @@ void BdaBridge<BridgeMatrix, BridgeVector, block_size>::solve_system([[maybe_unu
         if (h_rows.capacity() == 0) {
             h_rows.reserve(Nb+1);
             h_cols.reserve(nnzb);
-            getSparsityPattern(*mat, h_rows, h_cols);
+            getSparsityPattern(*mat, h_rows.data(), h_cols.data());
```

Comment:
This is undefined behaviour as there are no objects in `h_rows.data()` or `h_cols.data()` if you just use `vector<>::reserve(N)`.  We will have `vector<>::size() == 0`, but `vector<>::capacity() >= N`.  It is of course **likely** that this won't blow up given that these are `vector<double>`s, but if you really want to use `vector<>::data()` instead of passing a reference to the `vector` itself, then you must ensure that the vector is properly initialised.